source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,40 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template <typename Policy, typename InputIt, typename Size, typename OutputIt,
+        typename ValueType>
+    OutputIt
+    histogram(Policy&& exec, InputIt start, InputIt end, Size num_bins,
+        ValueType first_bin_min_val, ValueType last_bin_max_val, OutputIt histogram_first); // (1)
```

Comment:
A potential ambiguity: will values equal to `last_bin_max_val` be counted? The parameter name kind of suggests these will. but the math below tells the opposite.

I would probably not use the term "bin" at all, changing it everywhere to "interval". The parameters could then be named `first_interval_begin` and `last_interval_end`. Perhaps still ambiguous, because mathematically these are half-intervals; yet it sounds better to me.

Reply:
changed.